v1/api/swagger/components/schemas/VaAttachProtocol.yaml

```diff
@@ -0,0 +1,9 @@
+description: >-
+  Current volume attachment protocol expressed as a string enum value (iscsi, nvmetcp, fuse, fc)
```

Comment:
Hi @chitranm, have you looked at ProtocolParameters.yaml and ProtocolKind.yaml in the schemas directory? These seems to be the old way of specifying attach protocol? See NewVolumeAttachment.yaml. These all needs to be consistent.

Reply:
Hi @mchuang3, just now I was looking at quake code and found that Protocol related files (in hpegl-metal-client) are already present. I will revert this and modify them. 
Thanks.